tsdfileapi/api.py

```diff
@@ -945,7 +946,7 @@ def decrypt_nacl_headers(self, headers: tornado.httputil.HTTPHeaders) -> None:
         )
         self.nacl_chunksize = int(headers['Nacl-Chunksize'])
     
-    def additional_log_details(self) -> str:
+    def additional_log_details(self, separator: str = ", ") -> str:
```

Comment:
Likely it would be better if this function returns a dict instead, as it's going to be joined together with other strings later in `error_for_exception()`. That would allow setting the separator in only one location, and allow for different separators per exception type if needed.